simq/src/androidTest/java/com/simprints/simq/QualityWeightsTest.kt

```diff
@@ -0,0 +1,77 @@
+package com.simprints.simq
+
+import android.graphics.Bitmap
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.common.truth.Truth.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class QualityWeightsTest {
+    private lateinit var simQ: SimQ
+
+    @Before
+    fun setUp() {
+        simQ = SimQ()
+    }
+
+    private fun createTestBitmap(
+        width: Int = 256,
+        height: Int = 256,
+    ): Bitmap = Bitmap.createBitmap(width, height, Bitmap.Config.ARGB_8888)
+
+    @Test
+    fun defaultWeightsSumTo1() {
+        val weights = QualityWeights.DEFAULT
+        val sum =
```

Comment:
Unless there is a hidden logic that enforces weight sum to be 1, this is only testing addition in Kotlin :D